examples/python/human_pose_tracking/main.py

```diff
@@ -190,6 +190,23 @@ def get_downloaded_path(dataset_dir: Path, video_name: str) -> str:
     return str(destination_path)
 
 
+def get_blueprint() -> rrb.BlueprintLike:
+    return rrb.Viewport(
+        rrb.Horizontal(
+            rrb.Vertical(
+                rrb.Spatial2DView(origin="video", name="Segmentation/reprojection"),
```

Comment:
I find the slash in there a bit strange tbh - looks like an entity path again
What do you think of

```suggestion
                rrb.Spatial2DView(origin="video", name="Segmentation & reprojection"),
```

or just `Segmentation` or `Result`
```

Reply:
good point. let's go for `Result`